libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFUserAccountManagerTests.m

```diff
@@ -86,15 +87,20 @@ - (void)setUp {
     // Delete the content of the global library directory
     NSString *globalLibraryDirectory = [[SFDirectoryManager sharedManager] directoryForUser:nil type:NSLibraryDirectory components:nil];
     [[[NSFileManager alloc] init] removeItemAtPath:globalLibraryDirectory error:nil];
-
     // Set the oauth client ID after deleting the content of the global library directory
     // to ensure the SFUserAccountManager sharedInstance loads from an empty directory
     self.uam = [SFUserAccountManager sharedInstance];
-    self.uam.oauthClientId = @"fakeClientIdForTesting";
+    [SFAuthenticationManager sharedManager].oauthClientId = @"fakeClientIdForTesting";
 
     // Ensure the user account manager doesn't contain any account
+    NSArray *userAccounts = [[SFUserAccountManager sharedInstance] allUserAccounts];
```

Comment:
@khawkins Updating this method is ok if we move this method to internal, else it may result in  unintended consequences if used.  

The previous description 

```
/** Clear all the accounts state (but do not change anything on the disk).
 */
- (void)clearAllAccountState;
```